src/commands/utility/playstore.js

```diff
@@ -0,0 +1,47 @@
+const { CommandStructures, SwitchbladeEmbed, MiscUtils, Constants } = require('../../')
+const { Command, CommandParameters, StringParameter } = CommandStructures
+const gplay = require('google-play-scraper')
+
+module.exports = class PlayStore extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'playstore'
+    this.aliases = ['googlestore']
+    this.category = 'utility'
+    this.parameters = new CommandParameters(this,
+      new StringParameter({ missingError: 'commands:playstore.noCountryProvided' }),
+      new StringParameter({ full: true, missingError: 'commands:playstore.noAppProvided' })
+    )
+  }
+  run ({ t, author, channel }, country, appName) {
+    channel.startTyping()
+    gplay.search({ term: appName, country: country, num: 8 }).then((apps) => {
+      const listEmbed = new SwitchbladeEmbed(author)
+        .setTitle(t('commands:playstore.results', { appName }))
+        .setDescription(`**${t('commands:playstore.sendNumber')}**\n\n\`[1]\` ${apps[0].title}\n\`[2]\` ${apps[1].title}\n\`[3]\` ${apps[2].title}\n\`[4]\` ${apps[3].title}\n\`[5]\` ${apps[4].title}\n\`[6]\` ${apps[5].title}\n\`[7]\` ${apps[6].title}\n\`[8]\` ${apps[7].title}`)
+      channel.send(listEmbed).then(async (msg) => {
```

Comment:
may use async?

Reply:
https://github.com/SwitchbladeBot/switchblade/wiki/Command-Guidelines#use-async-and-await-instead-of-then-whenever-possible-example

Reply:
Ok, fixed.